canonical-web-and-design/jaas-dashboard
# Enable panel tables on model details page

## Done

- Enable panel tables on model details page
- Change active status dot to green
- Add a loading spinner to model details page
- Conditionally render model details tables based on data available

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click on a model
- Click the first app
- Observe app panel open
- You can now navigate the tables to slice and dice across apps/units and machines
- Refresh the page
- Observe the loading spinner

## Details

Related: https://github.com/canonical-web-and-design/juju-squad/issues/1388
